### PR TITLE
Tslint enable unused no unused variable warning

### PR DESCRIPTION
**Pull request by blair**

### CODE REVIEW

File: src/exchanges/ccxt/index.ts [hunk 1]
```diff
@@ -391,7 +391,7 @@ export default class CCXTExchangeWrapper implements PublicExchangeAPI, Authentic
         const sourceSymbol = await this.getSourceSymbol(symbol);
         try {
             const rawTrades = await this.instance.fetchTrades(sourceSymbol, since, limit, params);
-            return rawTrades.map(({info, id, timestamp, datetime, symbol: _symbol, order, type, side, price, amount}) => ({
+            return rawTrades.map(({id, timestamp, symbol: _symbol, side, price, amount}) => ({
```
Comment (fb55) [inline src/exchanges/ccxt/index.ts]: `_symbol` is a bit misleading here, even though it was already like this.

**Comment by fb55** [comment]
The rebase introduced a new issue :/

File: src/exchanges/ccxt/index.ts (hunk 1 above)
Comment (blair) [reply src/exchanges/ccxt/index.ts]: @fb55 Thanks, good point. Renamed _symbol to productId to match where it is used below.

**Comment by blair** [comment]
@fb55 Rebased and fixed the new issue. PTAL.

**Comment by fb55** [comment]
Nice, thanks!

**Pull request merged**